integration/data/test_snapshot.py

```diff
@@ -85,6 +85,11 @@ def test_snapshot_revert_with_backing_file(backing_dev):  # NOQA
     before = read_dev(dev, offset, length)
     assert before != ""
 
+    info = cmd.snapshot_info()
```

Comment:
Add the test to the same commit that made the change.

The other one is fine since 1. it has the unit test 2. it needs one additional patch to test correctly.